### PR TITLE
created v2 for index actions, no authentication there

### PR DESCRIPTION
**Pull request by justynapaldyna**
i think it works :D

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -37,7 +37,7 @@ group :development, :test do
   gem 'factory_bot_rails'
   gem 'rails-controller-testing'
 end
-
+gem 'active_model_serializers', '~> 0.10.12'
```
Comment (paleksandrowicz) [inline Gemfile:40]: Good choice, but how are you using those serializers in the controllers?

Comment (justynapaldyna) [reply Gemfile:40]: Okkk, i included serializers in controllers, user "each_serializer: ExampleSerializer". When tried to do just "serializer: ..." it didnt work, showed me 'undefined method' :)

Comment (justynapaldyna) [reply Gemfile:40]: used*

File: app/controllers/v2/admin_controller.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class V2::AdminController < ApplicationController
+  
+    def users
+      users = User.all
+      render json: users
```
Comment (paleksandrowicz) [inline app/controllers/v2/admin_controller.rb]: use serializers here also please

File: app/controllers/v2/categories_controller.rb [hunk 3]
```diff
@@ -0,0 +1,6 @@
+class V2::CategoriesController < ApplicationController
+    def index 
+        categories = Category.all 
+        render json: categories
```
Comment (paleksandrowicz) [inline app/controllers/v2/categories_controller.rb]: use serializers here also please

File: Gemfile (hunk 1 above)
Comment (paleksandrowicz) [reply Gemfile:40]: With single object we use serializer: with arrays we use each_serializer:

Comment (justynapaldyna) [reply Gemfile:40]: aaa ok thanks i understood the opposite from documentation xD

**Pull request merged**